RULES/xilinx/bitstream.go

```diff
@@ -2,25 +2,34 @@ package xilinx
 
 import (
 	"fmt"
-	"time"
 
 	"dbt-rules/RULES/core"
 	"dbt-rules/RULES/hdl"
 	h "dbt-rules/hdl"
 )
 
+var SynthFlattenStrategy = core.StringFlag{
+	Name:        "flatten_hierarchy",
```

Comment:
This are global, so you should prefix them. I think we've so far used hyphens as separators. Ie:. "hdl-synth-flatten-strategy"

Reply:
probably the first component should be the package name? so `xilinx-synth-flatten-strategy`?